src/OrleansAWSUtils/Storage/Provider/DynamoDBStorageProvider.cs

```diff
@@ -86,9 +88,10 @@ public Task Init(string name, IProviderRuntime providerRuntime, IProviderConfigu
 
             isDeleteStateOnClear = config.Properties.ContainsKey(DELETE_ON_CLEAR_PROPERTY_NAME) &&
                 "true".Equals(config.Properties[DELETE_ON_CLEAR_PROPERTY_NAME], StringComparison.OrdinalIgnoreCase);
-
-            Log = providerRuntime.GetLogger("Storage.AWSDynamoDBStorage." + id);
-
+            var loggerFactory = providerRuntime.ServiceProvider.GetRequiredService<ILoggerFactory>();
+            var loggerName = $"{this.GetType().FullName}.{name}";
+            Log = new LoggerWrapper(loggerName, loggerFactory);
```

Comment:
similar comment that remained unaddressed before. Let's pass the logger (and loggerFactory) in the LoggerWrapper. No need to create an extra logger for it, as the infrastructure will optimize a few things for us